crux-mir/src/Mir/Language.hs

```diff
@@ -205,11 +208,13 @@ runTests (cruxOpts, mirOpts) = do
             setSimulatorVerbosity (Crux.simVerbose cruxOpts) sym
             let simCtx = C.initSimContext sym mirIntrinsicTypes halloc outH
                     C.emptyHandleMap mirExtImpl Crux.emptyModel
-            return $ Crux.RunnableState $
-                C.InitialState simCtx C.emptyGlobals C.defaultAbortHandler C.UnitRepr $
-                    C.runOverrideSim C.UnitRepr $ simTest symOnline fnName
+            return (Crux.RunnableState $
+                    C.InitialState simCtx C.emptyGlobals C.defaultAbortHandler C.UnitRepr $
+                     C.runOverrideSim C.UnitRepr $ simTest symOnline fnName
+                   , undefined
```

Comment:
A number of symbolic tests are failing on this `undefined`.  One example:
```
cd crux-mir
./translate_libs.sh
cabal new-run -- crux-mir test/symb_eval/num/checked_add.rs
```

Reply:
Oof. I didn't mean to leave that in there! It was a just a temporary "how many other compile errors are there?" workaround that I forgot I'd made.